src/Jobs/Recruit.Vacancies.Jobs/VacancyEvents/VacancyEventsJob.cs

```diff
@@ -28,9 +28,9 @@ public VacancyEventsJob(ILogger<VacancyEventsJob> logger, VacancyEventHandler va
                 
                 await UnpackAndExecute(eventItem.EventType, eventItem.Data);
             }
-            catch (Exception ex)
+            catch (JsonException ex)
```

Comment:
Worth adding a fallback catch Exception too?

Reply:
I put generic exception handling into the Handler class. It made it easier to log the type of event we were handling. This handler just catches any deserialisation exceptions.

Reply:
👍